src/AppInstallerCLICore/ExecutionReporter.cpp

```diff
@@ -54,6 +54,12 @@ namespace AppInstaller::CLI::Execution
 
     OutputStream Reporter::GetOutputStream(Level level)
     {
+        // If the level is not enabled, return a default stream which is disabled
+        if (!levelEnabled(level))

```

Comment:
```suggestion
        if (WI_IsFlagSet(m_enabledLevels, level))

```
LevelEnabled() function could be removed Or check the flag within LevelEnabled() function if you think we still need that function;